.env.production

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY=AIzaSyC-SEix9G59fJbyfI4ScMsRJs1D1JBqyS4
```

Comment:
This should probably be `.env.test` as we haven't created the production environment yet.